spec/data/conf/semester.yaml

```diff
@@ -3,4 +3,4 @@ semester:
   - start_at_week_nr: 33
     finish_at_week_nr: 4
   - start_at_week_nr: 5
-    finish_at_week_nr: 26
+    finish_at_week_nr: 32
```

Comment:
maybe use data/conf/semester.yaml also for testing. this file is pretty central and does not change a lot